tests/lib/cli-engine/cli-engine.js

```diff
@@ -3121,18 +3169,31 @@ describe("CLIEngine", () => {
                 `
             };
 
-            it("should lint only JavaScript blocks if '--ext' was not given.", () => {
-                CLIEngine = defineCLIEngineWithInMemoryFileSystem({
-                    cwd: () => root,
+            let cleanup;
+
+            beforeEach(() => {
+                cleanup = () => {};
+            });
+
+            afterEach(async () => {
+                await cleanup();
+            });
```

Comment:
In other places you use the simpler `afterEach(cleanup);` form, but this one was left as an explicit `async` after you removed the `cleanup = () => { };` line from its body in db6787f7478772d1c26d93fbba041dcde4a23585.

```suggestion
            afterEach(cleanup);
```